### DISCUSSION TITLE
Sub to my newsletter to recieve updates about me, if your interested!

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [TechStudent10/techstudent10#36] [General]
https://subtonewsletter.kendalldoescoding.gq/

**Comment by TechStudent10** [comment]
1. Not interested sorry
2. Maybe don't advertise in ANY Discussions tab

**Reply by KendallDoesCoding** [reply]
Sorry, I'm not advertising as of such, was just saying if your interested, you can sub to it!

**Reply by KendallDoesCoding** [reply]
not forcing you to.

**Reply by TechStudent10** [reply]
> Sorry, I'm not advertising as of such

Asking someone to just subscribe to a newsletter out of the blue is advertising.

**Reply by KendallDoesCoding** [reply]
Ok, sorry!
